### PR TITLE
Updating Xdebug docs.

### PR DESCRIPTION
**Pull request by JDDoesDev**
### CHANGES

* Updates the VSCode Xdebug documentation to use the appropriate variables for the workspace. @See https://code.visualstudio.com/docs/editor/variables-reference#_why-isnt-workspaceroot-documented
* Update the screenshot to reflect new settings
* Add optional Drush debugging settings
* Cleanup trailing whitespace

### CODE REVIEW

File: docs/content/tools/xdebug.md [hunk 1]
```diff
@@ -138,31 +138,43 @@ You can run your scripts in console and debug them in the same way as browser re
...
 
 You can debug both web requests and cli scripts using this configuration.
 
+### CLI Scripts: Drush {#vscode-drush}
+
+To debug Drush commands using Xdebug and VSCode add the following to your path mappings
+
+**Drush 8.x**
+
+- Add `"/usr/local/bin/drush": "${workspaceRoot}/bin/drush"` to `pathMappings` in your `launch.json` file.
```
Comment (achekulaev) [inline docs/content/tools/xdebug.md]: Is it `workspaceRoot` or `workspaceFolder`?

File: docs/content/tools/xdebug.md [hunk 2]
```diff
@@ -138,31 +138,43 @@ You can run your scripts in console and debug them in the same way as browser re
...
+
+**Drush 9.x**
+
+- Add `"/usr/local/bin/drush": "${workspaceRoot}/vendor/bin/drush"` to `pathMappings` in your `launch.json` file.
```
Comment (achekulaev) [inline docs/content/tools/xdebug.md]: Is it `workspaceRoot` or `workspaceFolder`?

Comment (JDDoesDev) [reply docs/content/tools/xdebug.md]: Fudge... missed the main reason I started this.  Good catch!

File: docs/content/tools/xdebug.md [hunk 3]
```diff
@@ -138,31 +138,43 @@ You can run your scripts in console and debug them in the same way as browser re
...
-    
+
 4. Set a breakpoint
 5. Select the **Listen for XDebug** configuration from the dropdown and click **Start Debugging**
 
 You can debug both web requests and cli scripts using this configuration.
 
+### CLI Scripts: Drush {#vscode-drush}
+
+To debug Drush commands using Xdebug and VSCode add the following to your path mappings under the configuration that begins with `"name": "Listen for XDebug",`
```
Comment (shelane) [inline docs/content/tools/xdebug.md]: please add comma: ... and VSCode, add the following...

Comment (JDDoesDev) [reply docs/content/tools/xdebug.md]: done!

**Pull request merged**